torch/csrc/jit/passes/quantization.cpp

```diff
@@ -151,6 +171,34 @@ class InsertObserversHelper {
   // record the current unique id used to avoid incrementing from 0
   // every time to find a unique id.
   int uid_ = 0;
+
+  // These are the IR patterns that we match when we insert observers.
```

Comment:
these are the patterns we match to skip inserting observers

Reply:
oh good point, thanks